src/EconomicStrategy/EconomicStrategyManager.ts

```diff
@@ -237,4 +244,64 @@ export class EconomicStrategyManager {
 
     return true;
   }
+
+  // tslint:disable-next-line:cognitive-complexity
+  private async smartGasEstimation(txRequest: ITxRequest): Promise<BigNumber | null> {
+    const normalizeWaitTimes = (stats: any) => {
```

Comment:
Move this method as class method, will reduce cognitive-complexity. Use type instead of `any`